Python/pystate.c

```diff
@@ -2977,6 +2977,26 @@ _none_shared(PyThreadState *tstate, PyObject *obj,
     return 0;
 }
 
+static PyObject *
+_new_bool_object(_PyCrossInterpreterData *data)
+{
+    if (data->data){
+        Py_RETURN_TRUE;
+    }
+    Py_RETURN_FALSE;
+}
+
+static int
+_bool_shared(PyThreadState *tstate, PyObject *obj,
+             _PyCrossInterpreterData *data)
+{
+    _PyCrossInterpreterData_Init(data, tstate->interp,
+            (void *) (Py_IsTrue(obj) ? 1ll : 0ll), NULL,
```

Comment:
Ah, that looked like `011` and `111`, which was a little confusing.  Is the suffix necessary?

Reply:
MSVC would normally raise a warning about an int being cast to a `void*`. Maybe CPython has that warning disabled, I'll try it

Reply:
llvm does as well

```default
Python/crossinterp.c:619:13: warning: cast to 'void *' from smaller integer type 'int' [-Wint-to-void-pointer-cast]
            (void *) (Py_IsTrue(obj) ? 1 : 0), NULL,
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 warning generated.
```

Reply:
```cpp
            (void *) (Py_IsTrue(obj) ? (uintptr_t) 1 : (uintptr_t) 0), NULL,
```
would accomplish the same thing. I haven't seen many suffixes in the CPython source code so this is probably a better option